code-journal.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>code journal</title>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" type="css" href="stylesheets/code-journal-styles.css">
+  </head>
+
+  <body>
+    <header>
+      <nav>
+        <ul>
+          <li><a href="index.html">index</a></li>
+          <li><a href="portfolio.html">portfolio</a></li>
+          <li><a href="about.html">about me</a></li>
+        </ul>
+      </nav>
+      <h1>suzanne harrison</h1>
+      <h2>tech life journal</n2>
+    </header>
+
+    <body>
+        <table>
+          <tr>
+            <td><h3 class="row-in-table">March 23, 2016</h3></td>
+          </tr>
+          <tr>
+            <td rowspan="4"><img class="photos" src="images/hamming.jpg" alt="nucleotide picture"></td>
+            <td><h3 class="boxtitletop">Hamming Distance Exercise</h3></td>
+          </tr>
+          <tr>
+            <td><strong>Exercise goal:</stong></td>
+          </tr>
+          <tr>
+            <td>calculating the Hamming distance (the number of differing characters) between two strings.</td>
+          </tr>
+          <tr>
+            <td><strong>Process toward solution:</stong></td>
+          </tr>
+          <tr>
+            <td><h3><button type="button" onclick="alert('Learn more about Hamming Distance') style="width:60px;height:100px;"><a class="portbottomlink" href="https://en.wikipedia.org/wiki/Hamming_distance">Learn more about Hamming Distance</a></h3></td>
```

Comment:
You have some inline styles here and a couple other places. Instead, try sizing these elements with CSS